sopel/modules/safety.py

```diff
@@ -141,12 +141,12 @@ def download_domain_list(bot: Sopel, path: str) -> bool:
     url = bot.settings.safety.domain_blocklist_url
     if url is None or not url.startswith("http"):
         url = "https://raw.githubusercontent.com/StevenBlack/hosts/master/hosts"
-    LOGGER.info("Downloading unsafe domain list from %s", url)
+    LOGGER.debug("Downloading unsafe domain list from %s", url)
```

Comment:
I'd keep that one as info, as it is a useful information at startup. That, or I'd add an INFO log in the setup hook function.

Reply:
I chose to add this in the setup function, leaving this more-detailed one for debug only.